README.md

```diff
@@ -51,7 +51,7 @@ Danger runs at the end of a CI build, she will execute a `Dangerfile`. This file
 :sparkles: | `lines_of_code` | The total amount of lines of code in the diff
 :pencil2:  | `files_modified` |  The list of files modified
 :ship: | `files_added` | The list of files added
-:recycle: | `files_removed` | The list of files removed
+:recycle: | `files_deleted` | The list of files removed
```

Comment:
This isn't me changing the name of it - this is me updating the READMe to the current one.